tests/application/test_format.py

```diff
@@ -0,0 +1,79 @@
+"""Tests for formatting application services."""
+
+import json
+from decimal import Decimal
+
+import pytest
+
+from parade.application.format import ProjectFormatter, format_as
+from parade.domain.activity import ActivityName, Duration, ScheduledActivity
+from parade.domain.project_network import ScheduledProjectNetwork
+
+
+@pytest.fixture
+def simple_scheduled_network() -> ScheduledProjectNetwork:
+    """Create a simple scheduled network for testing."""
+    activities = [
+        ScheduledActivity(
+            name=ActivityName("A"),
+            duration=Duration(Decimal(5)),
+            earliest_start=Duration(Decimal(0)),
+            earliest_finish=Duration(Decimal(5)),
+            latest_start=Duration(Decimal(0)),
+            latest_finish=Duration(Decimal(5)),
+        ),
+        ScheduledActivity(
+            name=ActivityName("B"),
+            duration=Duration(Decimal(3)),
+            depends_on=frozenset([ActivityName("A")]),
+            earliest_start=Duration(Decimal(5)),
+            earliest_finish=Duration(Decimal(8)),
+            latest_start=Duration(Decimal(5)),
+            latest_finish=Duration(Decimal(8)),
+        ),
+    ]
+    return ScheduledProjectNetwork(activities)
+
+
+class TestFormatAs:
+    """Tests for format_as verb."""
```

Comment:
```suggestion
class TestFormatAsJson:
```

These tests are focused on our current implementation and that's correct, let's make sure the name matches.